src/commcare_cloud/commands/terraform/entrypoint.tf.j2

```diff
@@ -108,10 +107,17 @@ module "commcarehq" {
   }
 
   account_alias = {{ (account_alias or "")|tojson }}
+}
```

Comment:
is there a stray unclosed bracket somewhere above?

Reply:
Nope, this is just the diff making it look more complicated than it is. Imagine that this line was a white `}` with out a +, and this line https://github.com/dimagi/commcare-cloud/pull/2237/files#diff-38c55dbcbe9e3c73f5b5c18b14b5aba1R122 were a green line with a plus. Then it might make more sense.